src/mavlink-router/endpoint.cpp

```diff
@@ -44,9 +44,8 @@
 
 #define UART_BAUD_RETRY_SEC 5
 
-Endpoint::Endpoint(const char *name, bool crc_check_enabled)
-    : _name{name}
-    , _crc_check_enabled{crc_check_enabled}
+Endpoint::Endpoint(const char *name)
+    : _name {name}
```

Comment:
extra space in the initialization bracket is not according to the coding style. Here and everywhere else